pyk/src/pyk/utils.py

```diff
@@ -78,6 +78,19 @@ def find_common_items(l1: Iterable[T], l2: Iterable[T]) -> Tuple[List[T], List[T
     return (common, newL1, newL2)
 
 
+def intersperse(iterable: Iterable[T], delimiter: T) -> Iterator[T]:
```

Comment:
Is this used downstream?

Reply:
Yes: https://github.com/runtimeverification/evm-semantics/blob/kevm-pyk-dist/kevm_pyk/src/kevm_pyk/solc_to_k.py#L253